app/post.go

```diff
@@ -859,7 +917,11 @@ func (a *App) DeletePost(postId, deleteByID string) (*model.Post, *model.AppErro
 	}
 
 	if err := a.Srv().Store.Post().Delete(postId, model.GetMillis(), deleteByID); err != nil {
-		return nil, err
+		var nfErr *store.ErrNotFound
```

Comment:
We don't need all this here,  it would be just `return nil, model.NewAppError(...`, right?

Reply:
Actually, I think this is changing the behavior. This was failing always previously, and now only fails if the error is a "not-found" error. Maybe we need the switch, but we need the default case too.

Reply:
yep you are right Jesús, it is a bug and I forgot to add the default case, thanks!

Reply:
Thanks Rodrigo. I am going to unresolve this so that it remains visible until a new commit resolves it.

Reply:
It is strange because I thought I already added it :thinking:

Reply:
It is already solved